CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [1.14.2](https://github.com/brave/brave-ios/releases/tag/v1.14.2)
+ 
+  - Added support for NFC authentication. ([#1609](https://github.com/brave/brave-ios/issues/1609))
+  - Restore lost bookmarks and favorites during 1.12 migration. ([#2015](https://github.com/brave/brave-ios/issues/2015))
```

Comment:
Maybe something along the lines of:

* Restoring lost bookmarks and favourites for a subset of users that occurred during the 1.12 migration.

It didn't affect every hence adding the` subset of users`.